docs/index.html

```diff
@@ -178,6 +178,12 @@ <h1>Design Tokens (legacy)</h1>
         </section>
 
         <section>
+          <div class="alert-banner">
+            <div class="icon">
+              <svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M10 0C4.486 0 0 4.486 0 10s4.486 10 10 10 10-4.486 10-10S15.514 0 10 0zM9 6a1 1 0 112 0v4a1 1 0 11-2 0V6zm1 9a1 1 0 100-2 1 1 0 000 2z"/></svg>
+            </div>
+            <p>These following tokens are out of date with Shopify's design system. Check out <a href="https://github.com/Shopify/polaris-react/blob/main/documentation/Color%20system.md">this page</a> for the up to date information.</p>
```

Comment:
super nit pick here but we could use `’` instead of `'`

```suggestion
            <p>These following tokens are out of date with Shopify’s design system. Check out <a href="https://github.com/Shopify/polaris-react/blob/main/documentation/Color%20system.md">this page</a> for the up to date information.</p>
```

Part of Polaris docs. Up to you if you want to make this change

`option + shift + ]` = `’`